src/ext/java/org/opentripplanner/ext/transmodelapi/TransmodelGraphQLPlanner.java

```diff
@@ -227,36 +265,38 @@ private RoutingRequest createRequest(DataFetchingEnvironment environment) {
             }
         }*/
 
-    if (request.vehicleRental && !GqlUtil.hasArgument(environment, "bikeSpeed")) {
+    if (preferences.rental().allow() && !GqlUtil.hasArgument(environment, "bikeSpeed")) {
```

Comment:
VehicleRentalRequest was part of StreetRequest before but I moved it out since it did not make sense to have 4 different rental requests in RoutingRequest. Currently there is no logic for having different rental settings for access/egress/direct etc. so does not make sense to have it grouped that way. 

Had I missed something? Do we want to change model in the future?